index.js

```diff
@@ -75,26 +61,26 @@ const renderSubMenu = (options, param2, index) => {
 
     subMenuElement.style.top = `calc(${index + 1} * ${elementHeight} + ${index + 1} * ${gapHeight})`
 
-    options.forEach(({ id, innerText, onclick }) => createElement("button", {
-        id,
+    options.forEach(({ innerText, onclick }) => createElement("button", {
         className: "sub-options",
         innerText,
         onclick
     }, subMenuElement))
 }
 
-
-// TODO: make 1 function from the 2
-const renderDigitalProgrammeList = () => {
+const renderNewScreen = () => {
     removeSubMenuElement()
-    handleOptionsElementOnClick("digitalProgrammeList")
-    console.log("Digital Programme List")
-}
 
-const renderDigitalEpg = () => {
-    removeSubMenuElement()
-    handleOptionsElementOnClick("digitalEpg")
-    console.log("Digital EPG")
+    mainElement.remove()
```

Comment:
you cannot remove main element as it is created in html.

If at some point your app requires main element to be not visible you have to options:
- create it in js (as before) OR
- implement showing/hiding it

Reply:
Solved